website/docs/registry/policy-libraries/publishing.mdx

```diff
@@ -0,0 +1,80 @@
+---
+page_title: Terraform Registry - Publishing Policy Libraries
+description: >-
+  How to publish Policy Libraries to the Terraform Registry.
+---
+
+# Publishing Policy Libraries
+
+A policy library is a collection of [Sentinel policies](https://docs.hashicorp.com/sentinel/writing) and [policy modules](https://docs.hashicorp.com/sentinel/extending/modules) that anyone can share through the Terraform Registry.
+
+Policy libraries are built and maintained on GitHub. Once a policy library has been published, you can release a new version of the policies by pushing a properly formatted Git tag, as described below.
+
+## Requirements for a Policy Library
+
+- **Public GitHub repository**: The files defining Sentinel policies and policy modules must be stored in a public GitHub repository.
+
+- **Repository name**: GitHub repositories containing Policy Libraries must be named according to the convention `policy-library-<NAME>`, where the `<NAME>` segment provides a name for your policy library.
```

Comment:
```suggestion
- **Repository name**: GitHub repositories containing Policy Libraries must be named according to the convention `policy-<NAME>-library`, where the `<NAME>` segment provides a name for your policy library.
```
From what I have experienced, the `<NAME>` segment needs to be between _policy_ and _library_